website/source/api/client.html.md

```diff
@@ -529,9 +529,9 @@ The table below shows this endpoint's support for
 [blocking queries](/api/index.html#blocking-queries) and
 [required ACLs](/api/index.html#acls).
 
-| Blocking Queries | ACL Required |
-| ---------------- | ------------ |
-| `NO`             | `none`       |
+| Blocking Queries | ACL Required        |
```

Comment:
Document the GC endpoint

Reply:
Added in https://github.com/hashicorp/nomad/pull/3343